changelog_unreleased/api/pr-8983.md

```diff
@@ -0,0 +1,13 @@
+#### ESM standalone bundles ([#8983](https://github.com/prettier/prettier/pull/8983) by [@Monchi](https://github.com/Monchi))
+
+Now standalone is also bundled as ESM. It works directly on browser.
+
+```js
+import prettier from "https://unpkg.com/prettier@latest/standalone";
+import parserGraphql from "https://unpkg.com/prettier@latest/parser-graphql";
```

Comment:
I don't think we can omit the extensions in browser, should use `https://unpkg.com/prettier/parser-graphql.mjs`.